### DISCUSSION TITLE
Idea for bookmarklet executor

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1068] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Recently,I discovered that some web URLs other that [the webstore](https://chromewebstore.google.com/) can affect Chrome extensions.
I'm asking if URLs like [Securly.com](Securly.com) can affect the securly extension.
If yes,we can create a DNS like `94.140.14.49` (bookmarklet executor DNS) but with [Securly.com](Securly.com).

**Comment by AshtonDavies** [comment]
Securly might, but I think at least GoGuardian allows its site to use the extension.

Bypassi has blogged about [using web-accessible resources listed in an extension's manifest file for script injection](https://blog.bypassi.com/_/swamp-ultra/).

Google Forms has access to Assessment Assistant, a hidden extension on managed profiles, which has access to the [Chrome Management API](https://developer.chrome.com/docs/extensions/reference/api/management), which can be used on extensions regardless of the LTBEEF patch.

**Reply by AshtonDavies** [reply]
I can set a rewrite on the DNS for something such as these if someone wants.

**Reply by SIMONHSCH** [reply]
are you doing with Securly ?

**Reply by Blobby-Boi** [reply]
the google forms semms interesting

**Reply by Blobby-Boi** [reply]
btw I think lightspeed might have one for it in this url
http://lsrelay-config-production.s3.amazonaws.com/f8db5b5547d93ef167b094ef5c73485f56e752bb4971141abb5cda8ffb88c8ee/block_screen.html

**Reply by Blobby-Boi** [reply]
I found it on google and its the old lightspeed blockpage url and it has access to the extension

**Reply by AshtonDavies** [reply]
![Screen Shot 2024-05-14 at 8 47 14 PM](https://github.com/3kh0/ext-remover/assets/117958718/a392661e-3da7-4f4f-b1fc-ca3b15be5a32)
Nothing in here I see is utilized by Lightspeed Filter Agent other than the update URL and literally any PNG file.

**Reply by SIMONHSCH** [reply]
My school doesn't use lightspeed.

**Reply by Blobby-Boi** [reply]
thats weird cuz if you have lightspeed installed then it can display the url that it blocked in that page but if you dont it cant.
